bundles/org.openhab.binding.pulseaudio/src/main/resources/OH-INF/i18n/pulseaudio.properties

```diff
@@ -75,7 +75,7 @@ thing-type.config.pulseaudio.source.simpleProtocolSourceFormat.option.s24be = PC
 thing-type.config.pulseaudio.source.simpleProtocolSourceFormat.option.s32le = PCM signed 32-bit little-endian
 thing-type.config.pulseaudio.source.simpleProtocolSourceFormat.option.s32be = PCM signed 32-bit big-endian
 thing-type.config.pulseaudio.source.simpleProtocolSourceIdleTimeout.label = Idle Timeout
-thing-type.config.pulseaudio.source.simpleProtocolSourceIdleTimeout.description = Timeout in ms after which the connection will be closed when no stream is running. This ensures that your speaker is not on all the time and the pulseaudio source can go to idle mode. -1 for no disconnection.
+thing-type.config.pulseaudio.source.simpleProtocolSourceIdleTimeout.description = Timeout in ms after which the connection will be closed when no stream is running. This ensures that your mic is not on all the time and the pulseaudio source can go to idle mode. -1 for no disconnection. 0 for immediate disconnection (recommended value, or expect strange behaviour !).
```

Comment:
I think we can put something more clarifying like "recommended value to avoid audio delay".

Reply:
Waiting for the feedback to this comment before merging.

Reply:
Indeed, it could be more clarifying. But it is not really an audio delay : it takes audio data from "before" the wanted data.
Maybe : "or expect irrevelant audio data capture"
(not satisfied by this either, I'm open to suggestion)

Reply:
Maybe "to avoid problems with buffered audio data" or "to avoid capturing buffered audio"?

Reply:
Thank you, feels better.
"to avoid capturing unwanted buffered audio"
Done